sympy/stats/crv_types.py

```diff
@@ -936,10 +936,10 @@ def Exponential(name, rate):
     >>> from sympy.stats import variance, std, skewness
     >>> from sympy import Symbol
 
-    >>> l = Symbol("lambda", positive=True)
+    >>> rate = Symbol("lambda", positive=True)
```

Comment:
It was not necessary to change this name in Examples though it does no harm.

Reply:
Just for consistency